packages/toolkit/config/postcss.config.js

```diff
@@ -1,10 +1,24 @@
 const path = require('path');
+const glob = require('fast-glob');
+
+const { getTenUpScriptsConfig } = require('../utils');
 
 module.exports = ({ file, env }) => {
+	const projectConfig = getTenUpScriptsConfig();
+	const { globalStylesDir, globalMixinsDir } = projectConfig.paths;
+
+	const globalCssFiles = glob.sync(`${globalStylesDir}**/*.css`);
+	const globalMixinFiles = glob.sync(`${globalMixinsDir}**/*.css`);
+
 	const config = {
 		plugins: {
 			'postcss-import': {},
-			'postcss-mixins': {},
+			'@csstools/postcss-global-data': {
+				files: globalCssFiles,
```

Comment:
What would files and mixins files be if there are no files? Should we only set this up if there are files?

Reply:
In my testing the value just is an empty array if there are no files. Which is the default value